docker-compose.yml

```
@@ -11,7 +11,7 @@ services:
     volumes:
       - .:/code
     ports:
-      - "8000:8000"
+      - "8000:8012"
```

```suggestion
      - "8012:8012"
```
Think it makes sense to just keep this all consistent